### PR TITLE
enable Ethernet

### PR DESCRIPTION
**Pull request by mistyron**
Ethernet PHY working.
For now, I would like to leave the tools I used during the bring up as part of the image, they can be removed later on (before we release to master, e.g. ) though. They're clearly sectioned off in the `bbappend`:
```
IMAGE_INSTALL_append = " phytool"
IMAGE_INSTALL_append = " ethtool"
IMAGE_INSTALL_append = " mdio-tools"
IMAGE_INSTALL_append = " net-tools"
IMAGE_INSTALL_append = " tcpdump"
IMAGE_INSTALL_append = " iputils"
```
Some (like `tcpdump`) we may choose to retain "forever" as it belongs to a standard toold case. @matinlotfali, your thoughts?

### CODE REVIEW

File: recipes-custom/xserver/xserver-xorg_1.20.8.bbappend
Comment (matinlotfali) [inline recipes-custom/xserver/xserver-xorg_1.20.8.bbappend:1]: Let's remove this

File: recipes-kernel/linux/linux-renesas_%.bbappend [hunk 1]
```diff
@@ -13,9 +12,11 @@ SRC_URI += "file://WIRELESS.cfg"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/dts:"
 SRC_URI += "file://0001-remove-sdhi1-uhs.patch"
+SRC_URI += "file://0002-add-vsc8531-userspace-dts.patch"
+SRC_URI += "file://0003-add-mistysom-phy-mode-change.patch"
+SRC_URI += "file://0004-remove-phy-interrupts-dts.patch"
```
Comment (matinlotfali) [inline recipes-kernel/linux/linux-renesas_%.bbappend:17]: Is it possible that you would combine all these patches into a single patch?

File: recipes-network/iputils/iputils_s20190709.bb
Comment (matinlotfali) [inline recipes-network/iputils/iputils_s20190709.bb:1]: This recipe is already present in `./poky/meta/recipes-extended/iputils/iputils_s20190709.bb`. 
So this file is not needed.

File: recipes-network/libpcap/libpcap_1.9.1.bb
Comment (matinlotfali) [inline recipes-network/libpcap/libpcap_1.9.1.bb:1]: This recipe is already present in `./poky/meta/recipes-connectivity/libpcap/libpcap_1.9.1.bb`.
So this file is not needed.

File: recipes-network/tcpdump/tcpdump_4.9.3.bb
Comment (matinlotfali) [inline recipes-network/tcpdump/tcpdump_4.9.3.bb:1]: This recipe is already present in `./meta-openembedded/meta-networking/recipes-support/tcpdump/tcpdump_4.9.3.bb`.
So this file is not needed.

File: recipes-network/mdio-tools/mdio-tools_1.2.0.bb
Comment (matinlotfali) [inline recipes-network/mdio-tools/mdio-tools_1.2.0.bb:1]: Why are mdio-tools recipes duplicated in `recipes-custom` and `recipes-network`?

**Review by matinlotfali — CHANGES_REQUESTED** [review]
Please see my comments

File: recipes-kernel/linux/linux-renesas_%.bbappend (hunk 1 above)
Comment (mistyron) [reply recipes-kernel/linux/linux-renesas_%.bbappend:17]: `0002` overrides the actual phy node,
`0003` edits the dtsi to set the phy-mode to `rgmii-id`
`0004` edits a different dtsi to remove the interrupt settings.
Would you suggest to combine them into a single file even though they di different things (to different files, as-is)?

Comment (matinlotfali) [reply recipes-kernel/linux/linux-renesas_%.bbappend:17]: I am also patching different files in `0001-remove-sdhi1-uhs.patch`, but I they were for the same purpose. I thought maybe it makes sense to bind patches together. As if it doesn't make sense to change one if the other is not also changing.

Comment (mistyron) [reply recipes-kernel/linux/linux-renesas_%.bbappend]: >As if it doesn't make sense to change one if the other is not also changing.

If you look at it this way, fair enough, point taken! I've merged the patches into a single file! Looks much cleaner :+1:

**Comment by matinlotfali** [comment]
@mistyron Let's merge develop into phy-enable-working2 so we could see a yocto build test.

File: recipes-kernel/linux/linux-renesas_%.bbappend (hunk 1 above)
Comment (mistyron) [reply recipes-kernel/linux/linux-renesas_%.bbappend]: > @mistyron Let's merge develop into phy-enable-working2 so we could see a yocto build test.

Ok, yeah I see you merrged the other PR, I'll pull, merge & push

**Review by matinlotfali — APPROVED** [review]

**Pull request merged**